dbt_platform_helper/utils/versioning.py

```diff
@@ -76,18 +77,18 @@ def get_github_released_version(repository: str, tags: bool = False) -> Tuple[in
     return parse_version(package_info["tag_name"])
 
 
-def get_app_versions():
+def get_platform_helper_versions():
     package_info = requests.get("https://pypi.org/pypi/dbt-platform-helper/json").json()
     released_versions = package_info["releases"].keys()
     parsed_released_versions = [parse_version(v) for v in released_versions]
     parsed_released_versions.sort(reverse=True)
+    version_from_file = Path(PLATFORM_HELPER_VERSION_FILE).read_text()
 
-    return parse_version(version("dbt-platform-helper")), parsed_released_versions[0]
-
-
-def get_file_app_versions():
-    version_from_file = Path(".platform-helper-version").read_text()
-    return parse_version(version("dbt-platform-helper")), parse_version(version_from_file)
+    return {
+        "package_version": parse_version(version("dbt-platform-helper")),
```

Comment:
I think this might be a more intuitive name...
```suggestion
        "local_version": parse_version(version("dbt-platform-helper")),
```